bin/hardening/ftp_client_not_installed.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# run-shellcheck
+#
+# CIS Debian Hardening
+#
+
+#
+# Ensure ftp client is not installed (Automated)
+#
+
+set -e # One error, it's over
+set -u # One variable unset, it's over
+
+# shellcheck disable=2034
+HARDENING_LEVEL=3
+# shellcheck disable=2034
+DESCRIPTION="Ensure ftp client is not installed"
+PACKAGE='ftp'
+
+# This function will be called if the script status is on enabled / audit mode
+audit() {
+    # 0 means true in bash
+    PACKAGE_INSTALLED=1
+
+    is_pkg_installed "$PACKAGE"
+    if [ "$FNRET" -eq 0 ]; then
+        # var will be reused in 'apply'
+        PACKAGE_INSTALLED=0
+        crit "$PACKAGE is installed"
+    else
+        ok "$PACKAGE is not installed"
+    fi
+
+}
+
+# This function will be called if the script status is on enabled mode
+apply() {
+    if [ "$PACKAGE_INSTALLED" -eq 0 ]; then
+        info "Removing $PACKAGE"
+        apt remove -y "$PACKAGE"
```

Comment:
apt is still not considered stable enough to be called non interactively

Reply:
probably because the func was not written at the time i wrote this script....
Probably will do a global pass in the future